File: src/test/java/net/dinkla/raytracer/samplers/NRooksTest.java
package net.dinkla.raytracer.samplers;

import org.junit.jupiter.api.BeforeEach;

public class NRooksTest extends GeneratorTest {

    @BeforeEach
    @Override
    public void initialize() {
        new NRooks().generateSamples(NUM_SAMPLES, NUM_SETS , samples);
    }
    
}

File: src/test/kotlin/net/dinkla/raytracer/samplers/NRooksTest.kt
package net.dinkla.raytracer.samplers

import org.junit.jupiter.api.BeforeEach

class NRooksTest : AbstractGeneratorTest() {

    @BeforeEach
    override fun initialize() {
        NRooks().generateSamples(NUM_SAMPLES, NUM_SETS, samples)
    }

}
